proto/substrait/algebra.proto

```diff
@@ -13,16 +13,22 @@ option go_package = "github.com/substrait-io/substrait-go/proto";
 option java_multiple_files = true;
 option java_package = "io.substrait.proto";
 
+// Common fields for all relational operators
 message RelCommon {
   oneof emit_kind {
+    // The underlying relation is output as is (no reordering or projection of columns)
     Direct direct = 1;
+    // Allows to control for order and inclusion of fields
     Emit emit = 2;
   }
 
   Hint hint = 3;
   substrait.extensions.AdvancedExtension advanced_extension = 4;
 
+  //Direct indicates no-op on ordering or presence of fields in the output
```

Comment:
no-op seems weird here. No change or something similar might be a better way to say this.

Reply:
ACK